### PR TITLE
Codestyle, better users, default host, README

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,13 @@ Example:
...
 ```machine jira.camptocamp.com login YOUR_USERNANE password YOUR_PASSWORD```
 
+Install it (`$HOME/.local/bin/` should be in your `PATH`):
+
+```python3 -m pip install --user --editable=.```
+
 Exemple of call:
 
-```venv/bin/python jisprint/get_single_sprint.py jira.camptocamp.com 1128```
+```get-single-sprint 1128```
```
Comment (gberaudo) [inline README.md:36]: The host has disappeared.

Comment (sbrunner) [reply README.md:36]: Yes, because of https://github.com/gberaudo/scrumtools/pull/6/commits/3d059387fb7ece3380e83e68e27c7e71d8f7af33 :-)

**Comment by sbrunner** [comment]
@gberaudo UP :-)

**Review by gberaudo — APPROVED** [review]
Thanks

**Pull request merged**